### PR TITLE
Analogue watch

### PR DESCRIPTION
**Pull request by nmoya**
Remember to install `adafruit_display_shapes` in the lib folder!

https://github.com/Stanko/retro-frame/assets/1767648/22d9ba5c-d89c-4c3b-bc84-d12e9e389ee2

### CODE REVIEW

File: src/utils.py [hunk 1]
```diff
@@ -16,6 +16,14 @@ def fps_from_filename(filename: str):
...
         return None
 
+def width_height_from_bitmap(bitmap):
+    # Detect sprite orientation
+    # if there is no size in the filename, each frame is considered to be a square
+    if (bitmap.height > bitmap.width):
+        return bitmap.width, bitmap.width
+    else:
+        return bitmap.height, bitmap.height
+
```
Comment (Stanko) [inline src/utils.py:26]: It seems you reverted the change I made earlier. Did I re-introduce a bug?

File: code.py [hunk 2]
```diff
@@ -43,8 +44,9 @@ def __init__(self, bmp_folder: str = "/bmp", gif_folder: str = "/gif", skip_conn
         # Dictonary order is irrelevant, it will be sorted when switching apps
         self.apps = {
             LoopImagesApp.name: lambda: LoopImagesApp(self.bmp_file_list, self.display),
-            ClockApp.name: lambda: ClockApp(self.display, self.network, self.real_time),
+            ClockApp.name: lambda: ClockApp(self.display, self.real_time),
             GifPlayerApp.name: lambda: GifPlayerApp(self.gif_file_list, self.display),
+            AnalogueClockApp.name: lambda: AnalogueClockApp(self.display, self.real_time),
```
Comment (Stanko) [inline code.py:49]: Nitpicking, and you don't have to touch it in this PR, but it bothers me a little that `self.display` is not always the first param 🙈

**Review by Stanko — APPROVED** [review]
Very nice! 🕙 

I would like to play with watch faces at some point

File: src/utils.py (hunk 1 above)
Comment (nmoya) [reply src/utils.py:26]: Unfortunately, if the bitmap filename does not contain the tile dimensions, the code was breaking.
dota-diretide.bmp is 32x256, without this code, the height will be set to 256.

Deleting only this function is also inconsistent with `frame_count_from_bitmap` that uses the same logic to count the number of tiles in the tilesheet.

**Pull request merged**